src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/dial_number_request_test.cc

```diff
@@ -180,6 +180,38 @@ TEST_F(DialNumberRequestTest, OnEvent_SUCCESS) {
   command->on_event(event);
 }
 
+MATCHER_P(CheckMobileResultAndInfo, result_code, "") {
+  const bool is_info_present =
+      (*arg)[am::strings::msg_params].keyExists(am::strings::info);
+  const bool is_result_code_correct =
+      result_code ==
+      static_cast<mobile_apis::Result::eType>(
+          (*arg)[am::strings::msg_params][am::strings::result_code].asInt());
+
+  return !is_info_present && is_result_code_correct;
+}
+
+TEST_F(DialNumberRequestTest, OnEvent_InfoIsNotString_InfoDropped) {
+  MessageSharedPtr command_msg(CreateMessage(smart_objects::SmartType_Map));
+  (*command_msg)[am::strings::params][am::strings::connection_key] =
+      kConnectionKey;
+  DialNumberRequestPtr command(CreateCommand<DialNumberRequest>(command_msg));
+  MockAppPtr mock_app(CreateMockApp());
```

Comment:
@ValeriiMalkov I have noticed that mock_app has the same creation process for each test, it's seems better to move this creation to SetUp method